net-misc/seafile-client/seafile-client-6.0.4-r1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Any reason not to remove -r0 while at it? I'd say this is safe enough patch to justify `git mv`.